File: C/test_HDC2080.c
#include "C_HDC2080.h"
#include <stdio.h>

int main()
{
	float temperature, humidity; 

	if(setup_hdc2080() != 0 ) return 0; 

	if(read_from_hdc2080(&temperature, &humidity)!=0) return 0;

	printf("{ temperature : %f, humidity : %f }\n", temperature, humidity);

	return 0;
}

File: Python/testHDC2080.py
#!/usr/bin/python
import sys
import Pi_HDC2080

hdc2000 = Pi_HDC2080.Pi_HDC2080()

print "{ \"temperature\" : %3.1f, \"humidity\" : %3.1f }" %(hdc2000.readTemperature(), hdc2000.readHumidity())

